operators/src/processing/raster_vector_join/mod.rs

```diff
@@ -93,7 +94,10 @@ impl VectorOperator for RasterVectorJoin {
             }
         );
 
-        let vector_source = self.sources.vector.initialize(context).await?;
+        // TODO: how to check the vector source? bevor init of the other sources?
```

Comment:
maybe allow passing a closure that checks the sources individually?

otherwise: don't use the `initialize_source` method but do it manually here.

Reply:
[7136c37](https://github.com/geo-engine/geoengine/pull/780/commits/7136c37923939b32082fb9d1c81bf81cb0785c08)